kostyor/rest_api.py

```diff
@@ -308,4 +309,7 @@ def cluster_list():
 
 
 if __name__ == '__main__':
+    conf.parse_args()
+    engine = create_engine(conf.CONF.database, convert_unicode=True)
```

Comment:
As you moved `db_session.remove()` out of `rest_api.py` can you also moved out `create_engine`? I think something like `db_api.configure_session(conf.CONF.database)` should work.